ToDoListWithOOP.py

```diff
@@ -0,0 +1,153 @@
+class Item:
+    def __init__(self, id, name, status):
+        self.id = id
+        self.name = name
+        self.status = status
+
+    def display_items(self):
+        if self.status==True:
+            symbol = "\u2714"
+
+        else:
+            symbol = "\u274C"
+        print(f"{self.id} {self.name} {symbol} ")
+
+
+
+class ToDoList:
+    def __init__(self):
+        self.items = {}
+
+    def add_item(self, id, item):
+        if id not in self.items:
+            self.items[id] = item
+        else:
+            print(f"Item with id {id} is already listed.")
+
+    def display_list(self):
+        for item in self.items.values():
+            item.display_items()
+        print("-" * 40)
+
+    def remeove_item(self, id):
```

Comment:
A simpler implementation which would avoid loop can be used here 
`
item = self.items.pop(id, None )
if item == None:
  print(....)
.....
`